src/pages/HomePage/pages/MarketPlace/components/forms/ProductForm.tsx

```diff
@@ -61,7 +44,7 @@ export function ProductForm({
                     label="Product name *"
                     id="product_name"
                     placeholder="Enter Product Name"
-                    className="col-span-2 w-full"
+                    className="col-span-1 w-full"
```

Comment:
and FormLayout should make `className="col-span-1 w-full"` redundant is something breaking?